checks/sast.go

```diff
@@ -61,36 +62,39 @@ func SAST(c *checker.CheckRequest) checker.CheckResult {
 	if sastScore != checker.InconclusiveResultScore &&
 		codeQlScore != checker.InconclusiveResultScore {
 		switch {
-		// sastScore >= codeQlScore only happens if:
-		// - sastScore is maximum and codeQl is enabled OR
-		// - sastScore is minimum and codeQl is not enabled
-		case sastScore == checker.MaxResultScore && sastScore >= codeQlScore:
-			return checker.CreateProportionalScoreResult(CheckSAST,
-				"all commmits are checked with a SAST tool", sastScore, checker.MaxResultScore)
-		case sastScore != checker.MaxResultScore && sastScore >= codeQlScore:
-			return checker.CreateMinScoreResult(CheckSAST,
-				"no SAST tool detected")
+		case sastScore == checker.MaxResultScore:
+			return checker.CreateMaxScoreResult(CheckSAST, "a SAST tool is run on all commits")
+		case codeQlScore == checker.MinResultScore:
+			return checker.CreateResultWithScore(CheckSAST,
+				checker.NormalizeReason("no SAST is run on all commits", sastScore), sastScore)
```

Comment:
slightly confusing to read.
maybe "SAST tool is not run on all commits" ?

Reply:
done